src/main/resources/config/liquibase/changelog/20170806142424_added_entity_Study.xml

```diff
@@ -58,6 +62,17 @@
             </column>
         </createTable>
 
+		<!-- New table for storing the list of participants that haven't receive a invitation email -->
+		<createTable tableName="havent_receive_email">
+            <column name="participants_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="studies_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+	
+		<addPrimaryKey columnNames="studies_id, participants_id" tableName="havent_receive_email"/>
         <addPrimaryKey columnNames="studies_id, participants_id" tableName="study_participant"/>
         
```

Comment:
Nit: column names should be singular - study_id and participant_id.

Also, is bigint required for storing participant and study id? int will allow for 2^32 values (over 4 billion records) and bigint just seems like a waste of space.